framework/xds_k8s_testcase.py

```diff
@@ -691,6 +691,84 @@ def assertAllBackendsReceivedRpcs(self, lb_stats):
                 msg=f"Backend {backend} did not receive a single RPC",
             )
 
+    def assertClientEventuallyReachesSteadyState(
+        self,
+        test_client: XdsTestClient,
+        rpc_type: str,
+        num_rpcs: int,
+        threshold_percent: int,
```

Comment:
Should we set the default `threshold_percent`? We only call it with `threshold_percent=1`